test/fixtures/parser/snowflake/snowflake_escape.sql

```diff
@@ -0,0 +1,2 @@
+-- Backslash escapes work in snowflake
+select 'c\' ' as escaped, 'c\' '' ' as escaped_double

```

Comment:
Is there a discrepancy between this test and the CHANGELOG? If I read this test case correctly, Snowflake dialect also needs to allow escaping single quotes by having two in a row `''`, but the CHANGELOG entry talks about fixing an issue with "backslash escaped single quoted strings", which would be `\'`.

Based on the change to the regex, do we need to test backlash followed by some other character?

Reply:
I don't *think* so - but see if my explanation below on why makes sense.